lms/static/js/discovery/main.js

```diff
@@ -0,0 +1,22 @@
+RequireJS.require([
+    'jquery',
+    'backbone',
+    'js/discovery/app',
+    'js/discovery/collection',
+    'js/discovery/form',
+    'js/discovery/result_list_view',
+    'js/discovery/filter_bar_view',
+    'js/discovery/search_facets_view'
+], function ($, Backbone, App, Collection, DiscoveryForm, ResultListView, FilterBarView, FacetsBarView) {
```

Comment:
@martynjames I'm not familiar enough with the standards for backbone apps...do they say anything about using the optimizer (best-practices, etc)?